utility/utils.py

```diff
@@ -11,6 +11,7 @@
 import traceback
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from munch import munchify
```

Comment:
`munch` is missing in requirements or setup files.

Reply:
Will take care of everything in next patch